src/main/java/org/folio/helper/CheckinReceivePiecesHelper.java

```diff
@@ -241,203 +211,11 @@ private void checkIfAllPiecesFound(List<String> pieceIds, List<Piece> pieces) {
-          addError(piece.getPoLineId(), piece.getId(), ITEM_NOT_RETRIEVED.toError());
-        });
-        return Collections.emptyList();
-      });
-  }
-
-  /**
-   * Checks if all expected piece records found in the storage. If any is
-   * missing, remove from piece reference to item and exclude piece from {@param piecesWithItems}
-   *
-   * @param expectedItemIds
-   *          list of expected item id's
-   * @param piecesWithItems
-   *          map with item id as a key and piece record as a value
-   * @param items
-   *          found item records
-   */
-  private void checkIfAllItemsFound(List<String> expectedItemIds, List<JsonObject> items,
-      Map<String, Piece> piecesWithItems) {
-    // Handle the case when for some reason some items are not found
-    if (items.size() < expectedItemIds.size()) {
-      List<String> foundItemIds = StreamEx.of(items).map(HelperUtils::extractId).toList();
-
-      expectedItemIds.stream()
-        .filter(id -> !foundItemIds.contains(id))
-        .forEach(itemId -> {
-          Piece piece = piecesWithItems.get(itemId);
-          piece.setItemId(null);
-          piecesWithItems.remove(itemId);
-        });
-    }
-  }
-
-  /**
-   * Collect all piece records with non-empty item ids. The result is a map with
-   * item id as a key and piece record as a value
-   *
-   * @param piecesGroupedByPoLine
-   *          map with PO line id as key and list of corresponding pieces as
-   *          value
-   * @return map with item id as key and piece record as a value
-   */
-  protected Map<String, Piece> collectPiecesWithItemId(Map<String, List<Piece>> piecesGroupedByPoLine) {
-    return StreamEx
-      .ofValues(piecesGroupedByPoLine)
-      .flatMap(List::stream)
-      .filter(piece -> StringUtils.isNotEmpty(piece.getItemId()))
-      .toMap(Piece::getItemId, piece -> piece);
-  }
-
-  /**
-   * @param item
-   *          inventory item
-   * @param piece
-   *          piece associated with the item
-   * @return future indicating if the item update is successful.
-   */
-  protected abstract Future<Boolean> receiveInventoryItemAndUpdatePiece(JsonObject item, Piece piece, RequestContext requestContext);
-
-  /**
-   * Updates piece records with receiving details which do not have associated
-   * item
-   *
-   * @param piecesGroupedByPoLine
-   *          map with PO line id as key and list of corresponding pieces as
-   *          value
-   * @return updated map passed as a parameter
-   */
-  protected abstract Map<String, List<Piece>> updatePieceRecordsWithoutItems(Map<String, List<Piece>> piecesGroupedByPoLine);
-
   /**
    * Stores updated piece records with receiving details into storage.
    *
-   * @param piecesGroupedByPoLine
-   *          map with PO line id as key and list of corresponding pieces as
-   *          value
+   * @param piecesGroupedByPoLine map with PO line id as key and list of corresponding pieces as
+   *                              value
```

Comment:
duplicate explnation of piecesGroupedByPoLine, comments is not helpful anymore. This simlar comments better to remove

Reply:
This changed is from first commit - nothing changed except that method was moved to make proper grouping